pageserver/src/page_service.rs

```diff
@@ -589,15 +628,17 @@ impl PageServerHandler {
         }))
     }
 
-    #[instrument(skip(self, timeline, req), fields(rel = %req.rel, blkno = %req.blkno, req_lsn = %req.lsn))]
+    #[instrument(skip(self, timeline, req, ctx), fields(rel = %req.rel, blkno = %req.blkno, req_lsn = %req.lsn))]
```

Comment:
Just FYI there's also a `skip_all` if you are going to skip all, and define custom fields anyway.

Reply:
Yeah, I tent to do that for new functions, but here, just cargo-culting on and on :)